harmoni_detectors/harmoni_stt/src/stt_google_service.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python3
+
+# Importing the libraries
+import rospy
+import roslib
+import numpy as np
+import os
+import io
+from google.cloud import speech_v1
+from google.cloud.speech_v1 import enums
+from audio_common_msgs.msg import AudioData
+from std_msgs.msg import String
+from harmoni_common_lib.constants import State, RouterDetector, RouterSensor
+from harmoni_common_lib.helper_functions import HelperFunctions
+from harmoni_common_lib.child import WebServiceServer
+from harmoni_common_lib.service_manager import HarmoniExternalServiceManager
+
+
+class STTGoogleService(HarmoniExternalServiceManager):
```

Comment:
This class has a ton in common with harmoni_detectors/harmoni_stt/src/w2l_service.py. They should probably each be extensions of some base STT class.

Reply:
I'll reorganize this as part of the next move to a harmoni_unit structure